src/app/components/Hits/Hits.jsx

```diff
@@ -138,7 +151,14 @@ class Hits extends React.Component {
         role="presentation"
       >
         {activeResultsCount}
+        <div className="nypl-clear-results">
```

Comment:
Can you make this into it's own separate component? Also, it should not display if there are no search queries, filters, or sort.